com.unity.template.renderstreaming-hd/Assets/Scripts/Signaling/FurioosSignaling.cs

```diff
@@ -28,15 +28,17 @@ public class FurioosSignaling : ISignaling
     {
         private float m_timeout;
         private bool m_running;
+        private SynchronizationContext m_mainThreadContext;
         private Thread m_signalingThread;
         private AutoResetEvent m_wsCloseEvent;
         private WebSocket m_webSocket;
 
         public delegate void OnSignedInHandler(ISignaling sender);
 
-        public FurioosSignaling(string url, float timeout)
+        public FurioosSignaling(string url, float timeout, SynchronizationContext mainThreadContext)
```

Comment:
Why don't you use the default parameter for `SynchronizationContext`?
Almost cases users indicate `SynchronizationContext.Current` for this argument.